hash/fmtowns_flop_orig.xml

```diff
@@ -1379,6 +1416,45 @@ Zurukamashi Ver 2.0                                                 Nichikonren
 		</part>
 	</software>
 
+	<software name="loop">
+		<description>:LOOP - Izanai no Kaikiten</description>
+		<year>1995</year>
+		<publisher>グローサー (Grocer)</publisher>
+		<info name="alt_title" value="：ＬＯＯＰ ～いざなひの回帰点～" />
```

Comment:
Should the description say “Izanahi no Kaikiten” to match the Japanese title, or is this an official Latin title?  I haven’t seen box art anywhere, and the title screen doesn’t seem to have Latin on it.

Reply:
There are no official Latin sources that I know of, but as I mentioned before, I'm pretty sure いざなひ is supposed to be a traditional spelling of the modern いざない, with the pronunciation unchanged. @0kmg seems to agree with using something that reflects the actual pronunciation instead of using a direct transliteration of the kana, but I'm pretty sure you both are more knowledgeable about Japanese than I am, so I'll go with whatever the "official" stance is on this.

Reply:
Yeah, the phonetic rather than lexical transliterations kind of bother me, but it is what MAME uses.